src/mf/mf-backend.cpp

```diff
@@ -105,6 +106,20 @@ namespace librealsense
             }
         }
 
+        wmf_hid_device::~wmf_hid_device()
+        {
+            try {
+                _connected_sensors.clear();
+                _opened_sensors.clear();
+                _streaming_sensors.clear();
+                CoUninitialize();
```

Comment:
The device-scoped `CoInitializeEx/CoUninitialize` will work if you avoid generating devices via `wmf_backend` class.
But in case you create objects with ` wmf_backend::create_hid_device` then the following flow will occur:
1. `CoInitializeEx()` in wmf_backend ctor
2. `CoInitializeEx()`  in wmf_hid_device ctor
3.  `CoUninitialize()` in wmf_hid_device dtor
This means that `CoUninitialize` will be invoked while the backend is still active, which may cause incorrect behavior
Let review this offline